ui/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/ManagementActor.scala

```diff
@@ -239,21 +239,23 @@ class ManagementActor(managers: ProcessingTypeDataProvider[DeploymentManager],
     processState match {
       case Some(state) if state.status.isFinished =>
         findDeployedVersion(idWithName).flatMap {
-          case Some(version) =>
-            deployedProcessRepository.markProcessAsCancelled(idWithName.id, version, Some("Scenario finished")).map(_ =>
+          case Some(version) => {
+            val FinishedDeploymentComment = DeploymentComment.unsafe("Scenario finished")
```

Comment:
It's not const, please change it to camelCase.

Reply:
changed, thx

Reply:
changed, thx